backend/followers/views.py

```diff
@@ -0,0 +1,174 @@
+            return response.Response(status=status.HTTP_401_UNAUTHORIZED)
+        else:
+            try:
+                author_id = get_author_id(request)
+                foreign_id = get_foreign_id(request)
+                
+                current_author = Author.objects.get(id = author_id)
+                foreign_author = Author.objects.get(id = foreign_id)
+                if foreign_author is not None:
+                    current_author.followers.add(foreign_author)
+                    followers = current_author.followers.all().order_by('displayName')
+                    followers_serializer = self.serializer_class(followers, many=True)
+                    result = {
+                        'result': "Foreign author add successfully",
+                        'followers': followers_serializer.data
+                    }
+                else:
+                    return response.Response("Error: Foreign author do not found", status=status.HTTP_404_NOT_FOUND)
+               
+                return response.Response(result, status=status.HTTP_200_OK)
+
+            except Exception as e:
+                return response.Response(f"Error: {e}", status=status.HTTP_404_NOT_FOUND)
+    def delete(self, request, author_id, foreign_author_id):
+        if check_author_id(request) == False:
+            return response.Response(status=status.HTTP_401_UNAUTHORIZED)
+        else:
+            try:
+                author_id = get_author_id(request)
+                foreign_id = get_foreign_id(request)
+                
+                current_author = Author.objects.get(id = author_id)
+                foreign_author = Author.objects.get(id = foreign_id)
+                if foreign_author is not None:
+                    current_author.followers.remove(foreign_author)
+                    followers = current_author.followers.all().order_by('displayName')
+                    followers_serializer = self.serializer_class(followers, many=True)
+                    result = {
+                        'result': "Foreign author delete successfully",
+                        'followers': followers_serializer.data
+                    }
+                else:
+                    return response.Response("Error: Foreign author do not found", status=status.HTTP_404_NOT_FOUND)
+               
+                return response.Response(result, status=status.HTTP_200_OK)
+
+            except Exception as e:
+                return response.Response(f"Error: {e}", status=status.HTTP_404_NOT_FOUND)
+
+class TrueFriendApiView(GenericAPIView):
+    """
+    URL: ://service/authors/{AUTHOR_ID}/friends
+    GET [local, remote] get all true friend of AUTHOR_ID
+    """
+    permission_classes = [IsAuthenticated]
+    serializer_class = FollowerSerializer
+    def get(self, request, author_id, foreign_author_id):
+        if check_author_id(request) == False:
+            return response.Response(status=status.HTTP_401_UNAUTHORIZED)
```

Comment:
Public method, don't need authentication.
```suggestion
    permission_classes = [AllowAny]
    serializer_class = FollowerSerializer
    def get(self, request, author_id, foreign_author_id):
        if check_author_id(request) == False:
            return response.Response(status=status.HTTP_404_NOT_FOUND)
```